VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/join-private-consultation/join-private-consultation.component.ts

```diff
@@ -18,12 +18,12 @@ export class JoinPrivateConsultationComponent {
     constructor(protected logger: Logger) {}
 
     roomsAvailable(): boolean {
-        return this.participants.some(p => p.current_room != null);
+        return this.roomDetails.length > 0;
     }
 
     getRoomDetails(): Array<any> {
         const currentRooms = this.participants
-            .filter(p => p.current_room != null)
+            .filter(p => p.current_room != null && !p.current_room.label.toLowerCase().includes('judgejoh'))
```

Comment:
can we change to .startsWith('participant') please :)
There are more room types coming up that they shouldnt be able to join through this UI